### PR TITLE
Features/plink2

### PR DESCRIPTION
**Pull request by northwestwitch**
Splitted Plink into subroutines and created tests. tested on rasta.

### CODE REVIEW

File: t/plink_sex_check.t [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+          . catfile(qw{ temp_directory $family_id _data }),
+    },
+    binary_fileset_prefix => {
+        input           => catfile(qw{ temp_directory $family_id _data }),
+        expected_output => q{--bfile}
+          . $SPACE
+          . catfile(qw{ temp_directory $family_id _data }),
+    },
+    sex_check_min_f => {
+        input           => q{family_id},
```
Comment (henrikstranneheim) [inline t/plink_sex_check.t]: This would be more relevant q{0.2 0.75}

File: t/plink_sex_check.t [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    },
+    read_freqfile_path => {
+        input           => catfile(qw{ temp_directory family_id _data.frqx }),
+        expected_output => q{--read-freq}
+          . $SPACE
+          . catfile(qw{ temp_directory family_id _data.frqx }),
+    },
+);
+
+my %specific_argument = ();
```
Comment (henrikstranneheim) [inline t/plink_sex_check.t]: This is more perlish: my %specific_argument;

File: t/plink_calculate_inbreeding.t [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+use lib catdir( dirname($Bin), q{lib} );
+use MIP::Script::Utils qw{ help };
+
+our $USAGE = build_usage( {} );
+
+my $VERBOSE = 1;
+our $VERSION = '1.0.0';
+
+## Constants
+Readonly my $SPACE               => q{ };
```
[the remark this answers is not in the document]
Comment (northwestwitch) [reply t/plink_calculate_inbreeding.t]: yes, I fix it!

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**